apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -25,37 +25,51 @@ const styles = {
   }
 };
 
-const list = [
-  ["Lesson", "1", "2", "3", "4"],
-  ["Level Type", "1", "2", "3", "4"],
-  ["Student 1"],
-  ["Student 2"],
-  ["Student 3"],
-  ["Student 4"],
-  ["Student 5"],
-  ["Student 6"],
-  ["Student 7"],
-  ["Student 8"],
-  ["Student 9"],
-  ["Student 10"],
-  ["Student 11"],
-  ["Student 12"],
-];
-
 const columnWidths = [150, 50, 100, 300, 50];
 
 export default class VirtualizedDetailView extends Component {
 
+  static propTypes = {
```

Comment:
not required for this PR, but once the structure of our data is totally settled, we should pull this prop shape out as a constant so we can import and use the same thing everywhere.